src/components/Author/styles.ts

```diff
@@ -2,15 +2,15 @@ import styled from 'react-emotion';
 import ListItem from '@material-ui/core/ListItem';
 import Typography from '@material-ui/core/Typography';
 
-export const Heading = styled(Typography)`
-  && {
-    font-weight: 700;
-    text-transform: capitalize;
-  }
-`;
-export const AuthorListItem = styled(ListItem)`
-  && {
-    padding-left: 0;
-    padding-right: 0;
-  }
-`;
+export const Heading = styled(Typography)({
+  '&&': {
+    fontWeight: 700,
```

Comment:
I think there is `fontWeight.bold` from `utils/styles/sizes`?

Reply:
I agree with you, @griffithtp, in all the comments said below. I will create a new issue to attach every of them, because I prefer to keep the PR in the scope of the issue, refactor CSS templates in JS objects.

If we apply some of these changes we will change the class hashes and the test snapshots need to be updated like I said before in the description of the PR.

I will update this comment with the reference to the issue :wink:

Reply:
https://github.com/verdaccio/ui/issues/107